src/docfx/Errors.cs

```diff
@@ -355,6 +355,9 @@ public static Error RedirectionUrlNotFound(string from, SourceInfo<string> sourc
                             "Replace the redirect URL with a valid absolute URL, or set redirect_document_id to false in .openpublishing.redirection.json.";
                 return new Error(ErrorLevel.Suggestion, "redirection-url-not-found", message, source);
             }
+
+            public static Error CircularRedirection(SourceInfo? source, IEnumerable<FilePath> redirectionChain)
```

Comment:
nit: reuse `CircularReference` as `FilePath.ToString()` already says it is a `redirection`?

Reply:
`CircurlarRefernce` is `Error`, and `CirclaurRedirection` is supposed to be `Warning`?

Reply:
Yeah, warning is better.